README.md

```diff
@@ -38,7 +38,7 @@ downloads.
 
 ## Git
 
-To download the very latest source off the Git server do this:
+To download the very latest source of the Git server do this:
```

Comment:
This changes the meaning of the sentence. The latest source of the server would imply you get a server source code downloaded. Downloading it *off* the server would imply you get it from the server.

Maybe use *from* instead of *off* ?